content/en/developers/integrations/agent_integration.md

```diff
@@ -440,6 +440,33 @@ sudo datadog-agent integration install -w /path/to/wheel.whl
 
 Once you have created your Agent-based integration, see the [Create a tile][20] documentation for information on populating the remaining [required assets][31] that appear on your integration tile, and opening a pull request.
 
+## Update your integration
+To update your integration, edit the relevant files and open a new pull request to your integration directory in the [integrations-extras][21] or [marketplace][22] repository. 
+
+If you are editing or adding new integration code, a version bump is required.
+
+If you are editing or adding new README content, manifest information, or assets such as dashboards and recommended monitors, a version bump is optional. 
+
+With or without a version bump, updates to assets such as dashboards and recommended monitors immediately become available to customers after the corresponding pull requests are merged and they are published. Updates to `README.md`, `manifest.json`, or any other non-code files also immediately become visible to customers after publishing. 
```

Comment:
Its not necessarily `immediate`, but there's indeed no customer or developer intervention needed to see these updates.  

```suggestion
With or without a version bump, updates to assets such as dashboards and recommended monitors automatically become available to customers after the corresponding pull requests are merged and they are published. Updates to `README.md`, `manifest.json`, or any other non-code files also immediately become visible to customers after publishing. 
```